app/main.py

```diff
@@ -1,3 +1,86 @@
+from typing import Union
```

Comment:
The syntax `int | float` for type hints is only valid in Python 3.10 and later. If you are targeting an earlier version, use `Union[int, float]` instead.